src/conversions.jl

```diff
@@ -5,6 +5,62 @@ export rescale
 
 using ..LeapSeconds
 
+"""
+   utctai(jd1, jd2)
+
+Transform a two-part Julia date from `UTC` to `TAI`.
+
+# Example
+
+```jldoctest
+julia> tai  = Epoch{TAI}(2.4578265e6, 0.30477440993249416)
+2017-03-14T07:18:20.325 TAI 
+julia> AstronomicalTime.Epochs.taiutc(tai.jd1, tai.jd2)
+(2.4578265e6, 0.30434616919175345)
+```
+"""
+function utctai(jd1, jd2)
+    jd3 = jd1 + jd2
+    ls = leapseconds(jd3)
+    dtat = ls/SECONDS_PER_DAY;
```

Comment:
Since `jd3` is not used anywhere else, I would suggest to remove the intermediate variable.